# Created at 2020-12-23T13:02:09.948
org.testng.TestNGException: 
Cannot instantiate class TestGame
	at org.testng.internal.ObjectFactoryImpl.newInstance(ObjectFactoryImpl.java:30)
	at org.testng.internal.ClassHelper.instantiateUsingDefaultConstructor(ClassHelper.java:447)
	at org.testng.internal.ClassHelper.createInstance1(ClassHelper.java:370)
	at org.testng.internal.ClassHelper.createInstance(ClassHelper.java:336)
	at org.testng.internal.ClassImpl.getDefaultInstance(ClassImpl.java:110)
	at org.testng.internal.ClassImpl.getInstances(ClassImpl.java:188)
	at org.testng.TestClass.getInstances(TestClass.java:94)
	at org.testng.TestClass.initTestClassesAndInstances(TestClass.java:79)
	at org.testng.TestClass.init(TestClass.java:71)
	at org.testng.TestClass.<init>(TestClass.java:36)
	at org.testng.TestRunner.initMethods(TestRunner.java:461)
	at org.testng.TestRunner.init(TestRunner.java:340)
	at org.testng.TestRunner.init(TestRunner.java:293)
	at org.testng.TestRunner.<init>(TestRunner.java:178)
	at org.testng.SuiteRunner$DefaultTestRunnerFactory.newTestRunner(SuiteRunner.java:603)
	at org.testng.SuiteRunner.init(SuiteRunner.java:194)
	at org.testng.SuiteRunner.<init>(SuiteRunner.java:125)
	at org.testng.TestNG.createSuiteRunner(TestNG.java:1230)
	at org.testng.TestNG.createSuiteRunners(TestNG.java:1209)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1058)
	at org.testng.TestNG.runSuites(TestNG.java:997)
	at org.testng.TestNG.run(TestNG.java:965)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
	at org.testng.internal.ObjectFactoryImpl.newInstance(ObjectFactoryImpl.java:23)
	... 29 more
Caused by: java.lang.NullPointerException: Cannot invoke "java.util.List.add(Object)" because "this.table" is null
	at cs.lab.Game.resetTable(Game.java:28)
	at cs.lab.Game.<init>(Game.java:17)
	at TestGame.<init>(TestGame.java:11)
	... 35 more


